client/my-sites/sharing/connections/services-group.jsx

```diff
@@ -1,50 +1,51 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	classNames = require( 'classnames' );
+import React, { Component, PropTypes } from 'react';
+import classNames from 'classnames';
+import { connect } from 'react-redux';
+import { filter } from 'lodash';
 
 /**
  * Internal dependencies
  */
-var Service = require( './service' ),
-	ServicePlaceholder = require( './service-placeholder' ),
-	SectionHeader = require( 'components/section-header' ),
-	utils = require( 'lib/site/utils' );
+import { getKeyringServices } from 'state/sharing/services/selectors';
+import QueryKeyringServices from 'components/data/query-keyring-services';
+import SectionHeader from 'components/section-header';
+import Service from './service';
+import ServicePlaceholder from './service-placeholder';
+import utils from 'lib/site/utils';
 
 /**
- * Module variables
+ * Module constants
  */
-var NUMBER_OF_PLACEHOLDERS = 4;
-
-module.exports = React.createClass( {
-	displayName: 'SharingServicesGroup',
-
-	propTypes: {
-		site: React.PropTypes.object,
-		user: React.PropTypes.object,
-		connections: React.PropTypes.object,
-		onAddConnection: React.PropTypes.func,
-		onRemoveConnection: React.PropTypes.func,
-		onRefreshConnection: React.PropTypes.func,
-		onToggleSitewideConnection: React.PropTypes.func,
-		initialized: React.PropTypes.bool,
-		services: React.PropTypes.array,
-		title: React.PropTypes.string.isRequired,
-		description: React.PropTypes.string
-	},
-
-	getDefaultProps: function() {
-		return {
-			onAddConnection: function() {},
-			onRemoveConnection: function() {},
-			onRefreshConnection: function() {},
-			onToggleSitewideConnection: function() {},
-			initialized: false
-		};
-	},
-
-	getEligibleServices: function() {
+const NUMBER_OF_PLACEHOLDERS = 4;
+
+class SharingServicesGroup extends Component {
+
+	static propTypes = {
+		site: PropTypes.object,
+		user: PropTypes.object,
+		connections: PropTypes.object,
+		onAddConnection: PropTypes.func,
+		onRemoveConnection: PropTypes.func,
+		onRefreshConnection: PropTypes.func,
+		onToggleSitewideConnection: PropTypes.func,
+		initialized: PropTypes.bool,
+		services: PropTypes.array,
+		title: PropTypes.string.isRequired,
+		description: PropTypes.string
+	};
+
+	static defaultProps = {
+		onAddConnection: function() {},
+		onRemoveConnection: function() {},
+		onRefreshConnection: function() {},
+		onToggleSitewideConnection: function() {},
+		initialized: false
+	};
+
+	getEligibleServices() {
```

Comment:
I think it makes sense to keep services separate from sites (i.e. nothing about service eligibility in the reducer), though it'd be fine to have a selector which performs the logic like that which exists here currently.